src/pages/Home.js

```diff
@@ -0,0 +1,40 @@
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function Home() {
+  return (
+    <>
+      <Heading>Find your perfect chair</Heading>
+      <MainContainer>
+        <StyledLink to={`./industrial`}>Industrial</StyledLink>
+        <StyledLink to={`./scandinavian`}>Scandinavian</StyledLink>
+        <StyledLink to={`./contemporary`}>Contemporary</StyledLink>
+        <StyledLink to={`./mid-century-modern`}>Mid-Century Modern</StyledLink>
+      </MainContainer>
+    </>
+  );
+}
+
+const Heading = styled.header`
```

Comment:
I would suggest to use a `h1` here instead of a `header`. A header is a sectioning element, so you would use it if you have multiple elements you wanna group together. :)